pkg/netceptor/ping.go

```diff
@@ -7,8 +7,21 @@ import (
 	"time"
 )
 
+// NetceptorForPing should include all methods of Netceptor needed by the Ping function
+type NetceptorForPing interface {
+	ListenPacket(service string) (PacketConner, error)
+	NewAddr(target string, service string) Addr
+	NodeID() string
+	Context() context.Context
+}
+
 // Ping sends a single test packet and waits for a reply or error.
 func (s *Netceptor) Ping(ctx context.Context, target string, hopsToLive byte) (time.Duration, string, error) {
+	return Pinger(s, ctx, target, hopsToLive)
```

Comment:
The `er` postfix is used for interfaces in golang, we will need to change this function name to avoid confusion.